windows/deploy_windows.ps1

```diff
@@ -55,17 +55,30 @@ Function Clean-Build-Environment
 
 # For sourceforge links we need to get the correct mirror (especially NISIS) Thanks: https://www.powershellmagazine.com/2013/01/29/pstip-retrieve-a-redirected-url/
 Function Get-RedirectedUrl {
-
     param(
         [Parameter(Mandatory=$true)]
-        [String]$URL
+        [string] $url
     )
 
-    $request = [System.Net.WebRequest]::Create($url)
-    $request.AllowAutoRedirect=$true
-    $response=$request.GetResponse()
-    $response.ResponseUri.AbsoluteUri
-    $response.Close()
+    foreach ($attempt in 1, 2, 3) {
```

Comment:
That’s a strange syntax. Is there any reason why that’s not a standard for loop (doesn’t that also exist in PS?)

Reply:
Probably depends where one comes from. Python is foreach-centric... ;)
The alternative would be:

```suggestion
    for ($attempt = 1; $attempt -le 3; $attempt++) {
```

Not sure if it's better though.

Reply:
I think it’s more standard, but it seems to be more verbose. 
I‘d go with the foreach loop then.